docs/deployment/file-access-auth.md

```diff
@@ -138,7 +161,9 @@ The following table demonstrates which rule will be applied for operations on a
 
 Configured path globs represent a point in time. That is, path globs are expanded when a configuration is applied to generate the set of monitored paths. This is not a "live" representation of the filesystem. For instance, if a new file or directory is added that would match a glob after the configuration is applied, it is not immediately monitored.
 
-Within the main Santa configuration, the `FileAccessPolicyUpdateIntervalSec` key controls how often any changes to the configuration are applied as well as re-evaluating configured path globs to match the current state of the filesystem.
+Within the main Santa configuration, the `FileAccessPolicyUpdateIntervalSec` key controls how often any changes to the configuration are applied as well as re-evaluating configured path globs to match the current state of the filesystem. This has a minimum value of 15 seconds.
+
+`**` syntax is not supported, only `*` is supported for wildcards.
```

Comment:
It might make sense to rephrase this slightly differently to note what is supported. Maybe:
```suggestion
Glob pattern support is provided by the libc `glob(3)` function. Extended glob patterns, such as globstar (`**`), are not supported.
```

Reply:
@lid what do you think of @mlw's suggestion. 

I think it captures the intent pretty well.

Reply:
Thanks for the nudge, incorporated/committed the suggestion